### ISSUE TITLE
Add mpv folder to start menu folder

### ISSUE THREAD

**Issue by shinchiro** [rossy/mpv-install#10]
To mimic standard installation on windows. I found it can be useful. A folder can have:
- mpv shortcut
- open cmd in current folder
- mpv-uninstall.bat shortcut
- manual.pdf shortcut
- etc..

start menu folder located in ``%AppData%\Microsoft\Windows\Start Menu\Programs``

**Comment by rossy** [comment]
Agreed, though it's not easy to make a shortcut from a batch file with out-of-the-box Windows tools. (Maybe PowerShell could do it?)
